### ISSUE TITLE
Tomogram path in training_imgs.txt is ignored by visualize_3dhm.py

### ISSUE THREAD

**Issue by bwmr** [nextpyp/cet_pick#5]
Hi, 

When using a name different than `{image_name}.rec`, `visualize_3dhm.py` fails with a `FileNotFoundError`, even if the tomogram path is correctly specified in the training images text file:

> constructing 3D tomogram visualization for lamella02A_ts_001_ali....
> [...] 
> FileNotFoundError: [Errno 2] No such file or directory: 'data/tomo/lamella02A_ts_001_ali.rec'

**Comment by huangqinwen** [comment]
what's the command line input arguments for this?

**Comment by bwmr** [comment]
`python ~/Scripts/cet_pick/cet_pick/visualize_3dhm.py --input exp/simsiam2d3d/nosirt_dog26/all_output_info.npz --color exp/simsiam2d3d/nosirt_dog26/all_colors.npy --dir_simsiam exp/simsiam2d3d/nosirt_dog26/ --rec_dir data/tomo/`

as far as I can tell, you cannot pass the image list as an additional input argument

**Comment by bwmr** [comment]
Corresponding entry in the image text-file for training: 
```
image_name	tilt_path	rec_path	angle_path
lamella02A_ts_013_ali	data/tomo/lamella02A_ts_013_ali.mrc	data/tomo/lamella02A_ts_013_ali_full_rec.mrc	data/tomo/lamella02A_ts_013_ali.tlt

**Comment by huangqinwen** [comment]
yeah if you input the actual data directory did you encounter any error? We will add the image text-file!

**Comment by bwmr** [comment]
yes, I encounter the error above - I think the problem is that while training is allowed on custom-named tomograms, 3D-Visualisation is not. Thanks for adding the feature :)

**Comment by huangqinwen** [comment]
thanks! we added the image text file as input argument. To run it, simple use: ```python visualize_3dhm.py --input path_to/all_output_info.npz --color path_to/all_colors.npy --dir_simsiam path_to_output/ --image_txt train_img_txt.txt```
